# HG changeset patch
# User Juan Pablo Carbajal <ajuanpi+dev@gmail.com>
# Date 1521040352 -3600
#      Wed Mar 14 16:12:32 2018 +0100
# Node ID 0bb22c75c2d43c98a151b2fe46b8bd7fb9215b60
# Parent  a41e264609c0476b00db062459255a3fc1d0c83d
pkg.m: fix name of variable during update

diff --git a/scripts/pkg/pkg.m b/scripts/pkg/pkg.m
--- a/scripts/pkg/pkg.m
+++ b/scripts/pkg/pkg.m
@@ -577,17 +577,17 @@ function [local_packages, global_package
       else
         describe (files, verbose, local_list, global_list);
       endif
 
     case "update"
       installed_pkgs_lst = installed_packages (local_list, global_list);
       if (numel (files) > 0)
          update_lst = {};
-         installed_names = {installed_pkgs_list.name}';
+         installed_names = {installed_pkgs_lst.name}';
          for i = 1:numel (files)
            idx = find (strcmp (files{i}, installed_names), 1);
            if (isempty (idx))
              warning ("pkg: package %s is not installed - skipping update", files{i});
            else
              update_lst = { update_lst, installed_pkgs_lst{idx} };
            endif
          endfor
